src/components/ChatLog.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+
+const ChatLog = (props) => {
+    const chatComponentsArray = props.entries.map((entry) => {
+        return (
+            <ChatEntry 
+            key={entry.id} 
+            id={entry.id}
+            body={entry.body}
+            sender={entry.sender}
+            timeStamp = {entry.timeStamp}
+            liked = {entry.liked}
+            toggleLike = {props.toggleLike}
+            />
+    );
```

Comment:
Indentation needs to be cleaned up. In the future, when you work on React projects, linters will warn you when your indentation doesn't conform to style.

For example, lines 11-17 need to be indented one level deeper. and Line 19 needs to match the level of line 9